src/legacy/core_plugins/tile_map/public/plugin.ts

```diff
@@ -0,0 +1,76 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+import { DataSetup } from '../../data/public';
+import { VisualizationsSetup } from '../../visualizations/public';
+import {
+  CoreSetup,
+  CoreStart,
+  Plugin,
+  PluginInitializerContext,
+  UiSettingsClientContract,
+} from '../../../../core/public';
+
+import { LegacyDependenciesPlugin, LegacyDependenciesPluginSetup } from './shim';
+
+// @ts-ignore
+import { createTileMapFn } from './tile_map_fn';
+// @ts-ignore
+import { createTileMapTypeDefinition } from './tile_map_type';
+
+/** @private */
+interface TileMapVisualizationDependencies extends LegacyDependenciesPluginSetup {
+  uiSettings: UiSettingsClientContract;
+}
+
+/** @internal */
+export interface TileMapPluginSetupDependencies {
+  // TODO: Remove `any` as functionsRegistry will be added to the DataSetup.
+  data: DataSetup | any;
```

Comment:
I hadn't noticed this in the previous vega PR -- what was the reason for `any` here? Currently it should have the interface for `registerFunction` in `DataSetup['expressions']`.

Reply:
thanks

Reply:
@lukeelmers  Todo comment is actual: DataSetup['expressions'] type has only **run** and **ExpressionRenderer** 
Do you want me to fix it in a separate PR?

Reply:
Done